src/main/java/it/gov/pagopa/apiconfig/core/model/creditorinstitution/Station.java

```diff
@@ -47,4 +47,7 @@ public class Station {
   @Schema(required = true, description = "number version")
   @NotNull
   private Long version;
+
+  @Schema(description = "Describe the station connection's type, true synchronous, false asynchronous")
```

Comment:
`    @Schema(requiredMode = Schema.RequiredMode.NOT_REQUIRED)
`
add optional

Reply:
put it as read only
@Schema(accessMode = Schema.AccessMode.READ_ONLY)

Reply:
rename in `is_connection_sync`

Reply:
not required is the default value